Modules/_testcapimodule.c

```diff
@@ -573,19 +573,19 @@ static PyObject *
 test_get_type_name(PyObject *self, PyObject *Py_UNUSED(ignored))
 {
     PyObject *tp_name = PyType_GetName(&PyLong_Type);
-    assert(strcmp(PyUnicode_AsUTF8(tp_name), "int") == 0);
+    assert(PyUnicode_EqualToUTF8(tp_name, "int"));
```

Comment:
Even though it's still useful, this has nothing to do with the PR title and should go into a separate PR.